marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.tsx

```diff
@@ -108,6 +111,23 @@ const MappingCard: React.FC<Props> = (props) => {
         setOpenMappingSettings(true);
     }
 
+    const sortMappingByUpdated = (mapArray) => {
```

Comment:
Looks like we are creating same helper function in both mapping and load card components to sort the cards based on updated date. I am thinking that this will also be used for match, merge and Custom cards probably, so let's move it to a common place like utils/data-conversion or utils/conversionFunctions or any other curation related helper file within utils and then simply import & re-use it in all the required components respectively.

Reply:
Great point @xnikhil08. Moved the helper function into utils/conversionFunctions.tsx and imported it in the respective Load and Map card components